lab-kyle/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Kweezy Getting A Job",
```

Comment:
Remember, no caps and no spaces in name. Wasn't able to run "npm install".
https://docs.npmjs.com/files/package.json